Makefile

```diff
@@ -40,6 +40,26 @@ golang-docker: ## Builds Docker images for Go components using buildx
 			op-node op-batcher op-proposer op-challenger op-dispute-mon op-supervisor
 .PHONY: golang-docker
 
+layr-labs-docker-push: ## Builds and pushes op-batcher/node Docker image to ghcr.io
```

Comment:
Would be good to validate that pulls can be done across different OS architectures

Reply:
[18384a3](https://github.com/Layr-Labs/optimism/pull/41/commits/18384a3b6550745e5f1e1a3dc4b7f4255064b34e) and [531a91c](https://github.com/Layr-Labs/optimism/pull/41/commits/531a91c0e6ffdb82f76ad48efb17f692cb613bc4)

updated make command and ci to push multi-arch images